### DISCUSSION TITLE
zh/guide/api/apifox

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#296] [Announcements]
# zh/guide/api/apifox

前往https://alist-v3.apifox.cn (https://alist-v3.apifox.cn)查看在线API文档，并支持调试。 apifox

https://alist.nn.ci/zh/guide/api/apifox.html

**Comment by lee-cq** [comment]
帅，想这个很久了

**Reply by zhengquantao** [reply]
111

**Comment by xjxjin** [comment]
通过测试发现

获取已完成任务
/api/admin/task/upload/done

获取未完成任务
/api/admin/task/upload/undone

重试任务
/api/admin/task/upload/retry

清除已完成任务
/api/admin/task/upload/clear_done

清除已成功任务
/api/admin/task/upload/clear_succeeded

这几个api 返回值是
{'code': 200, 'message': 'success', 'data': []}
或者
{'code': 200, 'message': 'success', 'data': None}

无法调用成功

**Reply by xjxjin** [reply]
我的alist版本是
Built At: 2024-05-02 16:29:25 +0000
Go Version: go1.21.9 linux/amd64
Author: Xhofe <i@nn.ci>
Commit ID: 4cbbda8
Version: v3.34.0
WebVersion: 3.34.0
